ckanext/datastore/blueprint.py

```diff
@@ -292,6 +305,19 @@ def dump_to(
     elif fmt == 'xml':
         writer_factory = xml_writer
         records_format = 'objects'
+    # (canada fork only): psql dump format
+    elif fmt == 'sql':
+        datastore_uri = str(DatastoreBackend.get_active_backend()._get_write_engine().url)
+        # dump clean table, schema, and data.
+        # quote all for blank values.
+        cmd = subprocess.Popen(
```

Comment:
This part should be in the backends/postgres.py module instead of in a view so that other datastore back ends could have their own implementations.